### ISSUE TITLE
Add option to "Exclude muted chats in unread count"

### ISSUE THREAD

**Issue by EarthlingX** [blacksailer/depecher#23]
In big groups it is annoying that all new messages appear as unread messages in client. Maybe one could add an option to "Exclude muted chats in unread count" to disable their current behavior with getting displayed as new messages.

**Comment by blacksailer** [comment]
Hello!

Currently in unread count on cover summarizes values from unmuted chats AND all mentions in chats.

What do you mean exactly? 

Вторник, 24 апреля 2018 г получено от EarthlingX:
> In big groups it is annoying that all new messages appear as unread messages in client. Maybe one could add an option to "Exclude muted chats in unread count" to disable their current behavior with getting displayed as new messages.
> 
> -- 
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/blacksailer/depecher/issues/2

-- 
С уважением,
Черненький Иван

**Comment by EarthlingX** [comment]
Hi Blacksailer,

thanks a lot for your reply! :-)

So you say: If a chat is muted its new message won't be added to the "unread"-counter by default? So new messages are only included to the counter in case of unmuted chats? That would be great!

**Comment by blacksailer** [comment]
That already done...

I do not understand. You have muted chat, but unread messages from that chat is in unreac count in cover?

**Comment by chstem** [comment]
I would prefere to have muted chats included in the counter. I want to know wether there are new messages by locking at the cover, but I don't want my phone go haywire because some lively discussion is going in the int. group.

This is also the way Telegram Desktop is doing it. So this seems more konsistent me.

I know it is annoying when people request opposite things, but that is why we have settings.

**Comment by EarthlingX** [comment]
@blacksailer: You are right! This was a missunderstanding from my side. I'm sorry for this! Group chats are already disabled for counting in unread count. That was my dream. :-)
